onnxmltools/convert/sklearn/operator_converters/OneHotEncoder.py

```diff
@@ -11,9 +11,14 @@
 from ...common._registration import register_converter
 
 
-def convert_sklearn_one_hot_encoder(scope, operator, container):
+def convert_sklearn_one_hot_encoder(scope, operator, container, flag=False):
```

Comment:
Could we keep this module untouched if your OHE can be created with a few lines of code?